### DISCUSSION TITLE
Should this be reported as a bug?

### DISCUSSION THREAD

**Discussion by tobwen** [valhalla/valhalla#5013] [Q&A]
Is this a bug? Should it be reported?

https://valhalla.openstreetmap.de/directions?profile=car&wps=8.110933,49.198262,8.11169743537903,49.19840912477887

**Comment by chrstnbwnkl** [comment]
Looks like a bug to me. Seems to be only reproducible for that specific end coordinate, i.e. if you move it just slightly, it looks sane again. I wonder if it has anything to do with the node snapping.
